src/plugins/management/public/management_app.tsx

```diff
@@ -54,35 +54,38 @@ export class ManagementApp {
       title,
       mount: async ({}, params) => {
         let appUnmount: Unmount;
-        async function setBreadcrumbs(crumbs: ChromeBreadcrumb[]) {
-          const [coreStart] = await getStartServices();
-          coreStart.chrome.setBreadcrumbs([
-            {
-              text: i18n.translate('management.breadcrumb', {
-                defaultMessage: 'Management',
-              }),
-              href: '#/management',
-            },
-            ...crumbs,
-          ]);
-        }
-
-        ReactDOM.render(
-          <ManagementChrome
-            getSections={getSections}
-            selectedId={id}
-            legacySections={getLegacyManagementSections().items}
-            onMounted={async element => {
-              appUnmount = await mount({
-                basePath,
-                element,
-                setBreadcrumbs,
-              });
-            }}
-          />,
-          params.element
-        );
+        if (!this.enabledStatus) {
+          window.location.hash = '/management';
+        } else {
+          async function setBreadcrumbs(crumbs: ChromeBreadcrumb[]) {
```

Comment:
__optional nit:__ it was here before, so feel free to ignore, but technically we can move `setBreadcrumbs` function one level up to create it only once per app (or even make it a private class method assuming `getStartServices` can become a `private readonly` as well).

Reply:
I'll likely address this in a future refactor. Gave it a try but it twas changing a bit more code than I'd like for this PR.